packages/block-library/src/image/image-editing/use-transform-image.js

```diff
@@ -90,6 +91,15 @@ function useTransformState( { url, naturalWidth, naturalHeight } ) {
 		const el = new window.Image();
 		el.src = url;
 		el.onload = editImage;
+
+		const imgCrossOrigin = applyFilters(
+			'gutenberg.imageTransform.crossOrigin',
+			undefined,
+			url
+		);
+		if ( typeof imgCrossOrigin === 'string' ) {
```

Comment:
I think we could avoid this if by setting the return value of the filter directly to `crossOrigin` 🤔